### PR TITLE
Add setup instructions

### PR DESCRIPTION
**Pull request by rishotics**
- Add instructions to setup bundler locally
- Add instructions to run runop.ts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,78 @@
...
+- First clone the repo `https://github.com/eth-infinitism/bundler.git`.
+
+- Now install all the modules in the root directory `budler/` by doing:
+```shell 
+yarn
+```
+
+- Then we need to build individual packages and resolve dependancy issues. For this first create a file `testnet-mnemonic.txt` and paste your private key inside it. Now in the terminal:
+```shell
+MNEMONIC_FILE=testnet-mnemonic.txt yarn preprocess
```
Comment (plusminushalf) [inline README.md]: We can use `MNEMONIC_FILE=<path-to-mnemonic>`

File: README.md [hunk 2]
```diff
@@ -1,20 +1,78 @@
...
+  "beneficiary": "<YOUR_PUBLIC_KEY>",
+  "port": "3000",
+  "helper": "0x214fadBD244c07ECb9DCe782270d3b673cAD0f9c",
+  "entryPoint": "<YOUR_ENTRY_POINT_ADDRESS>",
+  "minBalance": "0",
+  "gasFactor": "1"
+}
+```
+
+-  As we have changed the mneumonic to private key, we need to make some changes to incoporate this:
```
Comment (plusminushalf) [inline README.md]: *mnemonic

File: README.md [hunk 3]
```diff
@@ -1,20 +1,78 @@
...
+    - We need to make changes in the `hardhat.config.ts` present inside `/bundler` package. Replace the `accounts` in `getNetwork` with:
+    ```shell
+        accounts: [mnemonic]
+    ```
+    - In the `packages/bundler/src/runBundler.ts` file we need to change two things. First, we should change the provider by giving a ethers JsonRpcProvider as follows:
+        ```shell
+        const provider: BaseProvider =
+            // eslint-disable-next-line
+            config.network === 'hardhat' ? require('hardhat').ethers.provider :
+            new ethers.providers.JsonRpcProvider(`https://NETWORK.infura.io/v3/YOUR_INFURA_ID`)
```
Comment (plusminushalf) [inline README.md:45]: I don't think we need this, I think we were failing mostly due to gas.

File: README.md [hunk 4]
```diff
@@ -1,20 +1,78 @@
...
+      ```
+      Now we need to change the Wallet which is extracted from mneunomic to a one extracted from private key:
+        ```shell
+        wallet = new Wallet(mnemonic, provider);
+        ```
+- Now you can start the bundler:
+```shell
+INFURA_ID=<YOUR_INFURA_ID> yarn run bundler --network goerli
```
Comment (plusminushalf) [inline README.md]: we can remove `--network goerli` since we have already added that in `bundle.config` above

Comment (rishotics) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (rishotics) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (rishotics) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (rishotics) [reply README.md:45]: I tried removing this, but the txn is failing

**Pull request merged**